lib/matplotlib/_mathtext_data.py

```diff
@@ -1066,7 +1068,7 @@
     'hermitmatrix'             : 8889,
     'barvee'                   : 8893,
     'measuredrightangle'       : 8894,
-    'varlrtriangle'            : 8895,
```

Comment:
This looks to have been correct? `lrtriangle` is U+25ff (9727) and `varlrtriangle` is U+22bf (8895).


reference: https://ctan.mirrors.hoobly.com/fonts/stix/doc/stix.pdf

Both are triangles, but stix assigns this the same as the old name